rules-reviewed/eap7/eap6/eap6-xml.windup.xml

```diff
@@ -168,6 +168,199 @@
+                        
+                       In the new implementation, the timestamp of a session is replicated on every request. 
+                       This prevents stale session meta data following failover.                        
+                    </message>
+                    <link href="https://docs.jboss.org/author/display/WFLY10/How+do+I+migrate+my+application+from+AS7+to+WildFly#HowdoImigratemyapplicationfromAS7toWildFly-WebSessionClustering" title="Web Session Clustering config migration changes in EAP 7" />
+                    <tag>cluster</tag>
+                    <tag>web</tag>
+                    <tag>configuration</tag>
+                </hint>
+            </perform>
+        </rule>    
+        <rule id="eap6-xml-08005">
+            <when>
+                <xmlfile matches="/*[local-name()='jboss-web']/*[local-name()='replication-config']/*[local-name()='snapshot-mode']" in="jboss-web.xml"/>
+            </when>
+            <perform>
+                <hint title="Web Session Clustering config snapshot-mode changes in EAP 7" effort="1" severity="optional">
+                    <message>
+                     The EAP 7 Web Session Clustering configuration deprecates `snapshot-mode` element in `jboss-web.xml` file descriptor.
+                     
+                     Previously, one could configure `snapshot-mode` as INSTANT or INTERVAL. Infinispan’s replication queue renders this configuration option obsolete.
+                    </message>
+                    <link href="https://docs.jboss.org/author/display/WFLY10/How+do+I+migrate+my+application+from+AS7+to+WildFly#HowdoImigratemyapplicationfromAS7toWildFly-WebSessionClustering" title="Web Session Clustering config migration changes in EAP 7" />
+                    <tag>cluster</tag>
+                    <tag>web</tag>
+                    <tag>configuration</tag>
+                </hint>
+            </perform>
+        </rule>    
+        <rule id="eap6-xml-08006">
+            <when>
+                <xmlfile matches="/*[local-name()='jboss-web']/*[local-name()='replication-config']/*[local-name()='snapshot-interval']" in="jboss-web.xml"/>
+            </when>
+            <perform>
+                <hint title="Web Session Clustering config snapshot-interval changes in EAP 7" effort="1" severity="optional">
+                    <message><![CDATA[
+                       The EAP 7 Web Session Clustering configuration changes `snapshot-interval` element meaning in `jboss-web.xml` file descriptor.
+                       Using `snapshot-interval` element is only relevant if there is used 
+                       
+                       ```xml
+                       <snapshot-mode>INTERVAL</snapshot-mode>
+                       ```
+                       
+                       ]]>
```

Comment:
I know that both https://docs.jboss.org/author/display/WFLY10/How+do+I+migrate+my+application+from+AS7+to+WildFly#HowdoImigratemyapplicationfromAS7toWildFly-WebSessionClustering and https://developer.jboss.org/wiki/ClusteringChangesInWildFly8 say "Only relevant for &lt;snapshot-mode>INTERVAL&lt;/snapshot-mode>. See above.", which is technically speaking correct, but can be confusing. The explanation here is downright misleading. We should say something like "This option was only relevant for &lt;snapshot-mode>INTERVAL&lt;/snapshot-mode>. Since &lt;snapshot-mode> is no longer used, this option is no longer used as well."